src/dotnet/CarbonAware.CLI/CarbonAwareCLI.cs

```diff
@@ -31,8 +36,8 @@ public CarbonAwareCLI(string[] args, ICarbonAwareAggregator aggregator)
         }
         catch (AggregateException e)
         {
-            Console.WriteLine("Error:");
-            Console.WriteLine(e.Message);
+            _logger.LogError("Error:");
+            _logger.LogError(e.Message);
```

Comment:
Is there a particular reason why we're splitting the logging statements instead of doing it one single one? (ala `logger.LogError(exception, "Error while processing request from {Address}", address)` for example)